docs/resources/dms_instance_v1.md

```diff
@@ -97,34 +95,35 @@ The following arguments are supported:
 * `subnet_id` - (Required) Indicates the ID of a subnet.
```

Comment:
Please add description that this is OTC subnet ID, equals to openstack network ID